### PR TITLE
first commit #93

### PR DESCRIPTION
**Pull request by clburns107**

### CODE REVIEW

File: app/views/viewings/new.html.erb [hunk 1]
```diff
@@ -1,12 +1,42 @@
...
+      </p>
+      <p>
+        <input type="date" class="form-datebox" name="date">
+      </p>
+      <p>
+        <button id="input_2" type="submit" class="form-submit-button">
+          SUBMIT
+        </button>
+      </p>
+    <% end %>
```
Comment (sumeetjain) [inline app/views/viewings/new.html.erb]: Remove trailing whitespace

File: app/views/viewings/new.html.erb [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+            </p>
+            <p>
+              <input type="date" class="form-textbox" name="date">
+            </p>
+            <p>
+              <button id="input_2" type="submit" class="form-submit-button">
+                SUBMIT
+              </button>
+            </p>
+          <% end %>
```
Comment (sumeetjain) [inline app/views/viewings/new.html.erb]: Remove trailing whitespace

File: app/views/viewings/new.html.erb [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+<head>
+    <title>New Viewing Form</title>
+</head>
+<body>
+  <div class="clear title">
+    <div class="col-12 title-box">
+      <h2>New Viewing Form</h2>
+    </div>
+  </div>
+
```
Comment (sumeetjain) [inline app/views/viewings/new.html.erb]: Remove trailing whitespace

File: app/views/viewings/new.html.erb [hunk 4]
```diff
@@ -1,25 +1,73 @@
...
-<% end %>  
+  <div class="clear content">
+    <div class="col-12 content-box">
+<!-- Checkbox div child of content div -->
+      <div class="checkbox">
+        <h2>Select a play to add to your viewings</h2>
+        <%= form_tag('/viewings/create/multiple') do %>
+        <% @unseen_plays.each do |play| %>
+          <%= check_box_tag "play_ids[]", play.id %>
+          <i class="play-title"><%= play.title %></i>
```
Comment (sumeetjain) [inline app/views/viewings/new.html.erb]: Remove trailing whitespace

**Comment by clburns107** [comment]
Im having trouble trying to figure out how to make the actual checkbox on this page larger... all the links relating to this on stack overflow seem to be down.

**Comment by sumeetjain** [comment]
Modifying form elements with CSS can be a huge pain–especially if you're trying to make them look the same across all browsers. 99% of the time, the best path is to _not_ modify them. So I'd recommend moving onto something else and coming back to this as a backlog item.

File: app/views/viewings/new.html.erb [hunk 5]
```diff
@@ -1,25 +1,79 @@
...
+
+
+      <div class="select-play">
+        <%= form_tag('/viewings/create/multiple') do %>
+        <% @unseen_plays.each do |play| %>
+          <p>
+            <%= check_box_tag "play_ids[]", play.id %>
+            <i class="play-title">
+              <%= play.title %>
+            </i>
```
Comment (sumeetjain) [inline app/views/viewings/new.html.erb]: Remove trailing whitespace

**Pull request merged**